CHANGELOG.md

```diff
@@ -42,16 +42,23 @@ You will only now need this:
 
 - Removes support for LibSass and Ruby Sass in favour of [Dart Sass](https://www.npmjs.com/package/sass)
 - Upgrade all dependencies to their latest versions where possible
-- Updates code formatting conventions to fall mostly in line with [Prettier](https://prettier.io/)'s recommendations. Have an opinion? [Share it!](https://github.com/nhsuk/nhsuk-frontend/discussions/827#discussioncomment-3188144) Details of the implementation and how it effects ESlint can be [found on the PR](https://github.com/nhsuk/nhsuk-frontend/pull/832#issue-1315246034).
+- Updates code formatting conventions to fall mostly in line with [Prettier](https://prettier.io/)'s recommendations. Details of the implementation and how it effects ESlint can be [found on the PR](https://github.com/nhsuk/nhsuk-frontend/pull/832#issue-1315246034).
+
+:wrench: **Fixes**
+
+- Adds missing `open` and `id` parameters to `details` nunjucks component #856  
```

Comment:
Hi - can we link to issue 856 here:
([Issue 856](https://github.com/nhsuk/nhsuk-frontend/issues/856))
Thanks!